demo/kubernetes/rook-storageclass.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: storage.k8s.io/v1beta1
+kind: StorageClass
+metadata:
+   name: rook-block
+   namespace: rook
+provisioner: kubernetes.io/rbd
+parameters:
+    monitors: 10.2.0.41:6790,10.2.1.68:6790,10.2.2.66:6790
+    adminId: admin
+    adminSecretName: rook-admin
```

Comment:
before you merge this, i would need to merge the operator changes that create these keys. But I'm not sure it makes sense to merge it when creating these demo artifacts by default. Let's talk about this flow and perhaps if it should be an api call instead